src/model_theory/terms_and_formulas.lean

```diff
@@ -369,13 +369,15 @@ end bounded_formula
 attribute [protected] bounded_formula.falsum bounded_formula.equal bounded_formula.rel
 attribute [protected] bounded_formula.imp bounded_formula.all
 
-localized "infix ` ≃ `:88 := term.bd_equal" in first_order -- input \~- or \simeq
-localized "infixr ` ⟹ `:62 := bounded_formula.imp" in first_order -- input \==>
-localized "prefix `∀'`:110 := bounded_formula.all" in first_order
-localized "prefix `∼`:max := bounded_formula.not" in first_order
+localized "infix ` ≃ `:88 := first_order.language.term.bd_equal" in first_order
```

Comment:
```suggestion
localized "infix ` =' `:88 := first_order.language.term.bd_equal" in first_order
```
I expect reusing the notation for `equiv` is not good for mathlib, even when localized. Something else is also fine.